src/lib/agent.c

```diff
@@ -64,6 +65,7 @@ static enum task_state validate(void *data)
 
 void sa_init(struct sof *sof, uint64_t timeout)
 {
+#if !CONFIG_AGENT_DISABLE
 	uint64_t ticks;
 
```

Comment:
@tlauda I think for now we can even entirely compile out agent.c and only provide a dummy sa_init() function when CONFIG_SOF_AGENT is not present.

In the future we can make an option to activate/disable the agent at runtime.

Reply:
@dbaluta Done.